### PR TITLE
Refactor folder structure

### LINKED ISSUE #255: Simpler folder structure

### PR DESCRIPTION
**Pull request by cstenglein**
Kinda big PR i know...

The easiest way would be to check out the changes and look at the structure instead of the changes itself.

fixes #255 

I wanted to reduce nesting & move the components where they actually belong. Took inspiration from [this video](https://www.youtube.com/watch?v=UUga4-z7b6s) & moved components which were not shared into the pages/<page> folder.

`containers` was removed and split up into the correct folders; `store` was renamed to `context` & created a `layouts` folder where the navigation / basic page layout resides.

Also created `__tests` folders to put all test files for that folder level inside.

Please give feedback if you don't like it / want something changed.

### CODE REVIEW

File: src/components/ConfirmModal.tsx [hunk 1]
```diff
@@ -2,10 +2,10 @@ import { FC, useContext } from "react";
...
 import { useTranslation } from "react-i18next";
 import { useNavigate } from "react-router";
-import ModalDialog from "../../../container/ModalDialog/ModalDialog";
-import { AppContext } from "../../../store/app-context";
-import { instance } from "../../../util/interceptor";
-import { MODAL_ROOT } from "../../../util/util";
+import ModalDialog from "../layouts/ModalDialog";
+import { AppContext } from "../context/app-context";
+import { instance } from "../util/interceptor";
+import { MODAL_ROOT } from "../util/util";
```
Comment (escapedcat) [inline src/components/ConfirmModal.tsx:8]: Later `util.ts` could be renamed to `index.ts` and import could just be `from "../util";`

**Review by escapedcat — APPROVED** [review]
Looks much clearer, nice! WIll watch the video later.

File: src/components/ConfirmModal.tsx (hunk 1 above)
Comment (cstenglein) [reply src/components/ConfirmModal.tsx]: Thank you! Done

**Comment by escapedcat** [comment]
Watched it, thanks!

**Pull request merged**